packages/zoe/src/contracts/multipoolAutoswap.js

```diff
@@ -139,10 +148,12 @@ export const makeContract = harden(
       const { poolHandle, tokenKeyword, liquidityKeyword } = liquidityTable.get(
         tokenBrand,
       );
-      return zcf.getCurrentAllocation(
-        poolHandle,
-        harden([tokenKeyword, CENTRAL_TOKEN, liquidityKeyword]),
-      );
+      const amountMathKeywordRecord = buildAmountMathKeywordRecord([
+        tokenKeyword,
+        CENTRAL_TOKEN,
+        liquidityKeyword,
+      ]);
+      return zcf.getCurrentAllocation(poolHandle, amountMathKeywordRecord);
```

Comment:
what if instead of an `amountMathKeywordRecord`, it's a brandKeywordRecord instead? That way we could reuse the `brandKeywordRecord` from the `instanceRecord` if we need to, or easily alter it to change the keywords?